src/utils/browser.ts

```diff
@@ -85,3 +85,17 @@ export function isGecko(): boolean {
     'CanvasCaptureMediaStream' in w,
   ]) >= 4
 }
+
+/**
+ * Checks whether the browser is based on Chromium version ≥86 without using user-agent.
+ * Doesn't checks that the browser is based on Chromium, there is a separate function for this.
+ */
+export function isChromium86OrNewer(): boolean {
```

Comment:
is it safe to call in browsers, other than Chrome?

Reply:
It's designed to be called and is called after checking that the browser is Chromium